networks/moonriver.md

```diff
@@ -18,23 +18,47 @@ In order to collect as much feedback as possible and provide fast issue resoluti
 Moonriver is scheduled to follow a [5-phase launch process](https://moonbeam.network/networks/moonriver/launch/). Currently, Moonriver is in Phase 0 of the launch process and has the following configurations:
 
 - Runs as a parachain connected to the Kusama relay chain.
-- Has an active set of {{ networks.moonriver.staking.max_collators }} collators, all hosted by PureStake on behalf of the Moonbeam team. There will be an initial collator election in Phase 1 to expand the collator set to parties outside of the Moonbeam team.
-- There are two RPC endpoints (hosted by PureStake). People can run full nodes to access their own private RPC endpoints.
+- Has an active set of {{ networks.moonriver.staking.max_collators }} collators, all hosted by PureStake on behalf of the Moonbeam Foundation. There will be an initial collator election in Phase 1 to expand the collator set to parties outside of the Moonbeam team
+- There are two RPC endpoints (hosted by PureStake). People can run full nodes to access their own private RPC endpoints
 
 ![Moonriver Diagram](/images/moonriver/moonriver-diagram.png)
 
 Some important variables to note include:
 
+### General Configurations
+
 | Variable              | Value                                       |
-|-----------------------|---------------------------------------------|
-| Minimum gas price     | {{ networks.moonriver.min_gas_price }} gsed |
-| Block time            | {{ networks.moonriver.block_time }} seconds |
-| Block gas limit       | {{ networks.moonriver.gas_block }}          |
-| Transaction gas limit | {{ networks.moonriver.gas_tx }}             |
-| RPC endpoint          | {{ networks.moonriver.rpc_url }}            |
-| WSS endpoint          | {{ networks.moonriver.wss_url }}            |
+|:---------------------:|:-------------------------------------------:|
```

Comment:
Thanks for fixing this! I noticed while working on the general Moonriver docs that all the tables needed to be centered

I shared a link in the developer relations channel for an extension that will make the tables prettier in case you're interested